simpleflow/execute.py

```diff
@@ -0,0 +1,193 @@
+import subprocess
+import functools
+import inspect
+import json
```

Comment:
Suggestion :  create a `compat` module and eventually check if `ujson` is installed ?